curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -225,19 +232,29 @@ void FuseReplyErrByErrCode(fuse_req_t req, CURVEFS_ERROR errcode) {
 }
 
 void FuseOpLookup(fuse_req_t req, fuse_ino_t parent, const char *name) {
+    clientOpMetric->opLookup.qps.count << 1;
+    clientOpMetric->opLookup.inflightOpNum << 1;
```

Comment:
maybe, we need an `InflightGuard` and implementation like this


```cpp
struct InflightGuard {
   InflightGuard(bvar::Adder<int64_t>* inflight) : inflight_(inflight) {
        (*inflight_) << 1;
   }

   ~InflightGuard() {
        (*inflight_) << -1;
   }

   bvar::Adder<int64_t>* inflight_;
};

```

Reply:
fixed